crates/fj-app/src/main.rs

```diff
@@ -45,18 +45,29 @@ fn main() -> anyhow::Result<()> {
 
     let args = Args::parse();
     let config = Config::load()?;
-
     let path = config.default_path.unwrap_or_else(|| PathBuf::from(""));
     let parameters = args.parameters.unwrap_or_else(Parameters::empty);
     let shape_processor = ShapeProcessor {
         tolerance: args.tolerance,
     };
 
-    let model = if let Some(model) = args.model.or(config.default_model) {
+    let path_of_model = path.canonicalize().unwrap_or_default();
+
+    let model = if let Some(model) =
+        args.model.or(config.default_model).as_ref()
+    {
         let mut model_path = path;
         model_path.push(model);
         Some(Model::from_path(model_path.clone()).with_context(|| {
-            format!("Failed to load model: {}", model_path.display())
+            if path_of_model.as_os_str().is_empty() {
+                format!(
+                    "Model is not defined, can't find model defined inside the default-model also, add model like \n cargo run -- -m {}", model.display()
+                )
```

Comment:
That error message could use some more detail. It doesn't say where it searched for the model, unlike the more detailed error message below.

Reply:
sure will update it with something bit more reasonable.